native/source/mixpanel/detail/worker.cpp

```diff
@@ -190,6 +198,46 @@ namespace mixpanel
             }
         }
 
+        int Worker::parse_www_retry_after(const nanowww::Response& response)
+        {
+            mixpanel->log(Mixpanel::LogEntry::LL_TRACE, "/track HTTP Response Headers: \n" + response.headers()->as_string());
+            mixpanel->log(Mixpanel::LogEntry::LL_TRACE, "/track HTTP Response Body: \n" + response.content());
```

Comment:
Unless you feel strongly about keeping these lines, I'd remove them. Remember that `mixpanel->log` isn't a macro, so all its arguments are going to get evaluated no matter whether tracing is on or not. So right now you're paying for the stringizing of `response.content()` every single time you enter the `parse_www_retry_after` function, even when tracing is off. That's a cost that the customer might not want to pay, IMHO.